test/file04.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { Theme } from '@xsite-ui/common.style';
```

Comment:
The import of 'Theme' is not used in the code. It could be removed to avoid unnecessary imports.